raft-tech/TANF-app
# Fix combo box label bug and bold labels on `/edit-profile`

fix combo box bug where label would not be read by screen readers, make EditProfile labels bold to match mocks

### This pull request changes...

The USWDS component was written to have the label as a sibling of the `combo box` component from USWDS. Normally React doesn't like this implementation but it was causing the explicit `for` attribute to not work. I moved the label to be a sibling and added a fragment.

On the mocks, the labels for each input should be bold. I'm not sure where that was lost but this adds that back in.


## FUNCTIONALITY

A screen reader should now read the label of the combo box correctly and all labels on `/edit-profile` should be bold.


## TO TEST

1. Go to `/edit-profile` and open up the screen reader.
2. Navigate through the inputs until you reach the combo box.
3. The screen reader should read `Associated State, Tribe or Territory...`

### This pull request is ready to merge when...

  - OpenAPI  
  - Readme
  - Entity Relationship Diagram (ERD)
